test/ERC721Drop.t.sol

```diff
@@ -977,6 +977,49 @@ contract ERC721DropTest is Test {
         assertEq(dummyRenderer.someState(), "");
     }
 
+    function test_SupplyRoyaltyPurchase() public setupZoraNFTBase(100) {
+        vm.startPrank(DEFAULT_OWNER_ADDRESS);
+        vm.expectRevert(IERC721Drop.InvalidMintSchedule.selector);
+        zoraNFTBase.updateRoyaltyMintSchedule(1);
+
+        zoraNFTBase.updateRoyaltyMintSchedule(5);
```

Comment:
what are these `1` and `5` values?  maybe they should come from constants with clear names